shape_completion_training/src/shape_completion_training/model/filepath_tools.py

```diff
@@ -33,7 +47,16 @@ def create_or_load_trial(group_name=None, trial_path=None, params=None, trials_d
         return create_trial('tmp', params, trials_directory, write_summary)
 
 
-def load_trial(trial_path=None):
+def load_trial(trial_path):
+    """
+    @param trial_path: full path or relative path from shape_completion_training/trials
+    @type trial_path: str
+    @return:
+    """
+    trial_path = pathlib.Path(trial_path)
+    if not trial_path.is_absolute():
```

Comment:
This way a user can specify a absolute path, or a relative path (from shape_completion/trials). We discussed that you like specifying full path so that terminal tab-completion works. I like specifying relative path. Best of both worlds